api/v1alpha1/rekor_types_test.go

```diff
@@ -188,7 +188,9 @@ var _ = Describe("Rekor", func() {
 					Expect(k8sClient.Create(context.Background(), &rekorInstance)).To(Succeed())
 					fetched := &Rekor{}
 					Expect(k8sClient.Get(context.Background(), getKey(&rekorInstance), fetched)).To(Succeed())
-					Expect(fetched.Spec).To(Equal(expectedRekorInstance.Spec))
+					Expect(fetched.Spec.Pvc.Name).To(Equal(expectedRekorInstance.Spec.Pvc.Name))
+					Expect(fetched.Spec.Pvc.Size).To(Equal(expectedRekorInstance.Spec.Pvc.Size))
+					Expect(fetched.Spec.RekorSearchUI.Enabled).To(BeTrue())
```

Comment:
Here you need to dereference the boolean pointer. My mistake in the earlier suggestion.
```suggestion
					Expect(*fetched.Spec.RekorSearchUI.Enabled).To(BeTrue())
```